gui/magic_texticons.gui

```diff
@@ -140,13 +140,3 @@ texticon = {
 		fontsize = 16
 	}
 }
-
-texticon = {
```

Comment:
so you moved from mana cost to just inactive aura ?

Reply:
It was actually always there, under that name.